src/main.rs

```diff
@@ -39,6 +39,7 @@ fn main() -> Result<(), Box<dyn Error>> {
     };
     let sentry = sentry::init(sentry::ClientOptions {
         transport: Box::new(curl_transport_factory),
+        release: sentry::release_name!(),
```

Comment:
how does sentry get this information? Ie, is it the version number, or...?

Reply:
yup, it grabs them from the cargo compile time env vars, `CARGO_PKG_NAME`@`CARGO_PKG_VERSION`

Reply:
ah, perfect. thanks